slim-java.sh

```diff
@@ -86,194 +87,176 @@ function get_vm_impl() {
 }
 
 # Strip debug symbols from the given jar file.
-function strip_debug_from_jar() {
+strip_debug_from_jar() {
 	jar=$1
-	isSigned=`jarsigner -verify ${jar} | grep 'jar verified'`
-	if [ "${isSigned}" == "" ]; then
+	isSigned=$( jarsigner -verify "$jar" | grep 'jar verified' )
+	if [ "$isSigned" = "" ]; then
```

Comment:
Note for when I get back around to this, it could probably be, thus saving a line and a variable
```
if ! ( jarsigner -verify "$jar" | grep 'jar verified' ); then
```